src/classification/segmented_audio_analysis.py

```diff
@@ -1,26 +1,27 @@
 #!/usr/local/bin/python3.6
 
-import os
-import sys
-import re
 import csv
 import json
+import os
+import re
 import subprocess
+import sys
 import tempfile
 from multiprocessing import Pool
 
-from essentia.standard import MonoWriter
-
 if __name__ == "__main__":
     sys.path.insert(0, os.path.abspath(__file__ + "../../../"))
 
-from utilities.get_song_id import get_song_id
-from utilities.filehandler.handle_path import get_absolute_path
-from utilities.filehandler.handle_audio import get_MonoLoaded_Song
-from similarity.split_song import split_song
-from classification.classifier.profile_data_extractor import get_classifier_data
-from classification.extractor.low_level_data_extractor import make_low_level_data_file
 from bpm.bpm_extractor import get_song_bpm
+from classification.classifier.profile_data_extractor import \
+    get_classifier_data
+from classification.extractor.low_level_data_extractor import \
+    make_low_level_data_file
+from essentia.standard import MonoWriter
```

Comment:
3rd party should be above local modules no? pep8?

Reply:
Nicely spotted, fixed :)